timescaledb/tutorials/analyze-intraday-stocks/fetch-and-ingest.md

```diff
@@ -0,0 +1,230 @@
+conn = psycopg2.connect(database=config.DB_NAME, 
+                        host=config.DB_HOST, 
+                        user=config.DB_USER, 
+                        password=config.DB_PASS, 
+                        port=config.DB_PORT)
+
+# column names in the database (pgcopy needs it as a parameter)
+COLUMNS = ('time', 'symbol', 'price_open', 'price_close', 
+           'price_low', 'price_high', 'trading_volume')
+
+# iterate over the symbols list
+for symbol in symbols:
+
+    # specify a time frame (max 24 months)
+    time_frame = range(1, 2) # (last 1 months)
+
+    # iterate over the specified time frame
+    for month in time_frame:
+
+        # fetch stock data for the given symbol and month
+        # using the function you created before
+        stock_data = fetch_stock_data(symbol, month)
+
+        # create a copy manager instance
+        mgr = CopyManager(conn, 'stocks_intraday', COLUMNS)
+
+        # insert data and commit transaction
+        mgr.copy(stock_data)
+        conn.commit()
+```
+This will start inserting data for each symbol, one month at a time.
+
+```
+time               |symbol|price_open|price_close|price_low|price_high|trading_volume|
+-------------------+------+----------+-----------+---------+----------+--------------+
+2021-06-16 20:00:00|AAPL  |    129.21|     129.21|    129.1|    129.21|         10194|
+2021-06-16 19:59:00|AAPL  |    129.22|     129.25|   129.15|    129.15|          3844|
+2021-06-16 19:58:00|AAPL  |    129.33|     129.33|    129.2|     129.2|          5240|
+2021-06-16 19:57:00|AAPL  |    129.32|     129.32|   129.32|    129.32|          1568|
+2021-06-16 19:56:00|AAPL  |    129.35|     129.36|   129.35|    129.35|          2417|
+2021-06-16 19:55:00|AAPL  |   129.385|    129.385|   129.35|    129.35|           434|
+2021-06-16 19:54:00|AAPL  |     129.4|     129.41|    129.4|     129.4|          5634|
+2021-06-16 19:53:00|AAPL  |    129.41|     129.43|    129.4|     129.4|          3265|
+2021-06-16 19:52:00|AAPL  |    129.42|     129.42|   129.41|    129.42|          1681|
+2021-06-16 19:51:00|AAPL  |    129.41|     129.42|   129.41|    129.42|          1530|
+2021-06-16 19:50:00|AAPL  |    129.41|     129.41|   129.41|    129.41|           680|
+2021-06-16 19:49:00|AAPL  |    129.42|     129.42|   129.41|    129.41|           651|
+2021-06-16 19:48:00|AAPL  |    129.38|     129.42|   129.38|    129.42|           520|
+2021-06-16 19:47:00|AAPL  |    129.39|     129.39|   129.39|    129.39|          2207|
+2021-06-16 19:46:00|AAPL  |    129.38|      129.4|   129.38|     129.4|          5871|
+2021-06-16 19:45:00|AAPL  |    129.39|     129.39|   129.39|    129.39|           845|
+```
+
+<highlight type="tip">
+Fetching and inserting intraday data can take a while, so if you want to see results quickly, limit the number of months to one and/or limit the number of symbols.
+</highlight>
+
+Hurray, plumbing work is done, [now let's explore the dataset!][explore]
```

Comment:
```suggestion
Hooray, the plumbing work is done, [now let's explore the dataset!][explore]
```

Not sure how well 'plumbing' is understood, but for this audience it's probably OK.